source/NVDAObjects/JAB/__init__.py

```diff
@@ -229,7 +229,12 @@ def _get_TextInfo(self):
 		return super(JAB,self).TextInfo
 
 	def _isEqual(self,other):
-		return super(JAB,self)._isEqual(other) and self.jabContext==other.jabContext
+		if not isinstance(other,JAB):
+			return False

```

Comment:
> Ok, I will remove it -- Note I copied the implementation of UIA(Window), which has the extra check too.

Oh, sorry, so it does. That one is redundant too. :(

> What do you think of the "try/except" around the jabcontext comparison. Is it necessary?

`JABContext.__eq__` really shouldn't ever throw an exception--it calls isSameObject, which doesn't report errors--but I guess having the try/except there probably doesn't hurt as far as being future proof.